logistics_planning_sale/models/sale_order_line.py

```diff
@@ -56,7 +60,7 @@ def _prepare_logistics_schedule(self):
             "origin": self.order_id.name,
             "company_id": self.company_id.id,
             "destination_partner_id": self.order_id.warehouse_id.partner_id.id,
-            "partner_id": self.order_partner_id.id,
+            "partner_id": self.order_partner_shipping_id.id,
```

Comment:
In this case, as partner shipping is not required in any sales line view, I shouldn't create a new field, because it could be simply obtained as follows (like `name` three lines above):
```suggestion
            "partner_id": self.order_id.partner_shipping_id.id,
```